docs/3.0/fields/boolean_group.md

```diff
@@ -64,22 +64,8 @@ The output value must be a hash as described above.
 ```
 
 
-:::warning
-The final value in the database column will be overwritten, The options will not be appended.
+:::info
+<VersionReq version="3.7.0" class="mt-2" />
 
-
-```json
-// this
-{
-  "feature_enabled": true,
-  "another_feature_enabled": false,
-  "something_else": "some_value" // this will disappear
-}
-
-// becomes
-{
-  "feature_enabled": true,
-  "another_feature_enabled": false,
-}
-```
+The keys not specified in the options will remain unaffected. Previously, it removed all unspecified keys.
```

Comment:
```suggestion
Before version 3.7.0 Avo would override the whole attribute with only the payload sent from the client.

```json
// Before update.
{
  "feature_enabled": true,
  "another_feature_enabled": false,
  "something_else": "some_value" // this will disappear
}

// After update.
{
  "feature_enabled": true,
  "another_feature_enabled": false,
}
```

Version 3.7.0 and up will only update the keys that you send from the client.


```json
// Before update.
{
  "feature_enabled": true,
  "another_feature_enabled": false,
  "something_else": "some_value" // this will be kept
}

// After update.
{
  "feature_enabled": true,
  "another_feature_enabled": false,
  "something_else": "some_value"
}
```
```